docs/config/lua/keyassignment/ActivateLastTab.md

```diff
@@ -0,0 +1,16 @@
+# ActivateLastTab
+
+Activate the last active tab index. If there is none, it will do
+nothing. If the tabs get moved around, it will move to the last
+tab index rather than the last tab itself.
```

Comment:
is that the desired behavior, or just simple to implement behavior?
If desired, then I'd suggest adding a brief example of what that means if the user reorganizes.

However, I think it isn't too much extra complexity to switch to the prior tab using its tabid with the inline suggestions!